<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditProfileWindow</class>
 <widget class="QDialog" name="EditProfileWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>530</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: white;
    padding: 6px 12px;
    border: 1px solid gray;
    border-radius: 4px;
}
QPushButton:hover {
    background-color: #dddddd;
}
QPushButton:pressed {
    background-color: #cccccc;
}
</string>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(254, 227, 220);
	font: 20pt &quot;Source Code Pro&quot;;
	border: 2px solid indianred;
    border-radius: 10px;
    padding: 8px 10px;
}</string>
   </property>
   <property name="text">
    <string>Edit Profile</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSurname">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>340</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: white;
    border: 1px solid lightgray;
    border-radius: 10px;
    padding: 8px 14px;
    font-size: 14px;
}
</string>
   </property>
   <property name="placeholderText">
    <string>Surname</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditBio">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>410</y>
     <width>301</width>
     <height>113</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
    background-color: white;
    border: 1px solid lightgray;
    border-radius: 10px;
    padding: 8px 10px;
    font-size: 14px;
}
</string>
   </property>
   <property name="placeholderText">
    <string>Short Bio...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonChooseAvatar">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>200</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Change Avatar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditName">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>270</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: white;
    border: 1px solid lightgray;
    border-radius: 10px;
    padding: 8px 14px;
    font-size: 14px;
}
</string>
   </property>
   <property name="placeholderText">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAvatar">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>65</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 60px;
border: 2px dashed gray;
background-color: #ffffff;
</string>
   </property>
   <property name="text">
    <string>Avatar</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Name">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>250</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="Surname">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>320</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Surname</string>
   </property>
  </widget>
  <widget class="QLabel" name="Bio">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>390</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Bio</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditProfileWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditProfileWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
